app/src/app/admin/articles/article/article.ts

```diff
@@ -58,10 +58,14 @@ namespace app.admin.articles.article {
                     }
                 },
                 resolve: /*@ngInject*/{
-                    article: (articleService:common.services.article.ArticleService, $stateParams:IArticleStateParams):common.models.Article | ng.IPromise<common.models.Article> => {
+                    article: (articleService:common.services.article.ArticleService, $stateParams:IArticleStateParams, userService:common.services.user.UserService):common.models.Article | ng.IPromise<common.models.Article> => {
 
                         if (!$stateParams.permalink || $stateParams.permalink == 'new'){
+                            let currentUser = userService.getAuthUser();
+
                             let newArticle = articleService.newArticle();
+                            newArticle._author = currentUser;
+                            newArticle.authorId = currentUser.userId;
```

Comment:
This logic should actually go into the `articleService.newArticle()` method as it's not specific to this resolve.